casanovo/casanovo.py

```
@@ -294,13 +303,24 @@ def train(
     output_root: Optional[str],
     verbosity: str,
     force_overwrite: bool,
+    load_all_states: Optional[bool],
```

Ruff (0.15.1)

[error] Boolean-typed positional argument in function definition

(FBT001)